app/models/place.rb

```diff
@@ -0,0 +1,9 @@
+class Place < ApplicationRecord
```

Comment:
Please add a model test for places -- shoulda-matchers can make this relatively painless:

```
  describe 'validations' do
    it { should validate_presence_of(:first_name) }
    it { should validate_presence_of(:last_name) }
    it { should validate_presence_of(:email) }
  end
```